feature-libs/my-account/organization/assets/public_api.ts

```diff
@@ -1 +1 @@
-export * from './translations/translations';
```

Comment:
Why such a change? This file contained both translations and translations chunks config,

Reply:
Actually the point here was to fit in with file name convention in Spartacus, but you're might be right with this one.